torch/csrc/jit/interned_strings.h

```diff
@@ -3,146 +3,252 @@
 #include <stdint.h>
 #include <string>
 #include <unordered_map>
+#include <algorithm>
 
 #include "torch/csrc/jit/generated/aten_interned_strings.h"
 
 namespace torch { namespace jit {
 
-// JIT symbols are synthetic operators that occur only in the JIT IR
+// Every symbol is classified in a namespace, specifying what kind of symbol it
+// is.  Unsigned char to ensure widening to unique_t (also an unsigned type)
+enum class SymbolNamespace : unsigned char {
+  onnx  = 'o',
+  prim  = 'p',
+  aten  = 't',
+  // NB: ONNX and ATen attributes all live in a global namespace, as
+  // their interpretation depends on the operator name (which is namespaced)
```

Comment:
Global namespace?

Reply:
I guess it's more accurate to say "unified" namespace